### DISCUSSION TITLE
current_date in sql editor is different supabase js return

### DISCUSSION THREAD

**Discussion by ghost** [supabase/supabase#11350] [Questions]
Sorry about my english.
I am from Vietnam.
In sql editor, I set the timezone to Vietnam
```
alter database postgres
set timezone to 'Asia/Ho_Chi_Minh';
```
and i created a view
```
create view v_current_date as select current_date
```
i run in sql editor and result is "2022-12-30" (it's today in vietnam)
but when i run supabase-js 
```
const { data, error } = await supabase
          .from("v_current_date")
          .select("*")
          .single();
console.log(data);
```
and result is yesterday in vietnam
```
{
  "current_date": "2022-12-29"
}
```
how can i get the same result as sql editor?

**Comment by steve-chavez** [comment]
Hello, sorry for the late reply.

Can you try doing:

```sql
alter role authenticator set timezone to 'Asia/Ho_Chi_Minh';
-- then restart the db
```

Then retry the query with supabase-js.
